analysis/tvla.py

```diff
@@ -488,7 +510,7 @@ def run_tvla(ctx: typer.Context):
                              trace_start=trace_start, trace_end=trace_end)
 
                 if ((save_to_disk_trace is True or save_to_disk_ttest is True) and
-                        general_test is True and i_step == 0):
+                        (general_test) and i_step == 0):
```

Comment:
Do you need the parenthesis around `general_test` here?

Reply:
No. I've removed them now